GameFiles/IW4x/userraw/scripts/_integration.gsc

```diff
@@ -737,6 +808,18 @@ GotoPlayerImpl( target )
     self IPrintLnBold( "Moved to " + target.name );
 }
 
+PlayerToMeImpl( target )
+{
+    if ( !IsAlive( target ) )
+    {
+        self IPrintLnBold( target.name + " is not alive" );
+        return;
+    }
+
+    self SetOrigin( target GetOrigin() );
```

Comment:
This would call garbage collisions which is why this command doesn't need to exist imo. Really consider not adding it.